packages/editor/src/components/block-list/style.scss

```diff
@@ -305,6 +305,11 @@
 			border-bottom: $border-width solid $light-gray-500;
 			bottom: auto;
 		}
+
+		// Clear floats of centered block when previous sibling block has left/right float.
+		& + .editor-block-list__block[data-align="center"] {
```

Comment:
This is not loaded in the frontend. Is this the expectation or are we expected to load a similar style in the frontend too?

Reply:
Yeah this is the expectation, as themes will most likely already have code to deal with this on the frontend. Here is TwentyNineteen's: https://github.com/WordPress/twentynineteen/blob/d6f9595f376f45d8ba0f203d9c3e3a393a4b3c18/sass/modules/_alignments.scss

That theme clears all blocks aligned to center:

`.aligncenter { 
clear: both;
display: block;
margin-left: auto;
margin-right: auto;
}`

But I wasn't sure if being this opinionated inside the editor was desired, which is why the PR only targets centered blocks that come after a left/right aligned block. 

Do you think it should be more opinionated than that?

Reply:
This sounds correct I think. In the editor, the center align has to be independent of any floats, otherwise it doesn't make much sense, since it will obey to its parent container anyway, if there is any advanced layout. 

Currently it really makes no sense:

<img width="535" alt="Screenshot 2019-04-22 at 21 31 16" src="https://user-images.githubusercontent.com/107534/56517613-05be8980-6546-11e9-88e8-3987d12fc2d1.png">